taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    LENGTH = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number", )
+
+    def clean_license_number(self) -> str:
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != DriverLicenseUpdateForm.LENGTH:
+            raise ValidationError(
+                f"License number should be "
+                f"{DriverLicenseUpdateForm.LENGTH} symbols"
+            )
+
+        if not license_number[:3].isupper():
+            raise ValidationError("First 3 characters should be capitalized")
+
+        if not license_number[:3].isalpha():
+            raise ValidationError("First 3 characters should be letters")
+
+        if not license_number[-5:len(license_number):1].isdigit():
+            raise ValidationError("Last 5 characters should be digits")
+
+        return license_number

```

Comment:
Still no validation for DriverCreation"?

Reply:
![4324234234](https://user-images.githubusercontent.com/113552706/229571429-12ee6100-a55d-4d40-8c0a-76c20a44b36f.PNG)
as i understand all i had to do was use this form for driver creation view as i did, and i cant change this form name since it fails tests if i do, should i create separate creation form validator?

Reply:
Just create an additional `DriverCreationForm` which inherits from `UserCreationForm`. And implement validation on creation